eth/downloader/downloader.go

```diff
@@ -369,22 +371,7 @@ func (d *Downloader) Synchronise(id string, head common.Hash, td *big.Int, mode
 		}
 		return err
 	}
-
-	switch err {
```

Comment:
why not adding the `errInvalidChain` to this switch? it was an upstream change? (it's more readable as a switch in my opinion)